pkg/registry/apis/folders/register_test.go

```diff
@@ -121,20 +118,21 @@ func TestFolderAPIBuilder_Validate_Create(t *testing.T) {
 		},
 	}
 
-	s := (grafanarest.Storage)(nil)
-	m := &mock.Mock{}
-	us := storageMock{m, s}
-
-	b := &FolderAPIBuilder{
-		gv:         resourceInfo.GroupVersion(),
-		features:   nil,
-		namespacer: func(_ int64) string { return "123" },
-		folderSvc:  foldertest.NewFakeService(),
-		storage:    us,
-	}
```

Comment:
^^^ moved inside each test case.  odd/dangerous to share the same mock across each test